antispam_bee.php

```diff
@@ -396,7 +396,7 @@ private static function _init_internal_vars()
 				'dnsbl'		=> __( 'DNSBL Spam', 'antispam-bee' ),
```

Comment:
Could have been seperated. I agree. When we change the key, we get a problem with old data, where the spam reason is saved as `dnsbl`, which will not be recongnized no more, when I see this correctly?